aws-athena/views/qc-vw_iasworld_change_in_low_value_sales.sql

```diff
@@ -0,0 +1,39 @@
+WITH sales_cte AS (
+    SELECT
+        SUBSTR(sales.saledt, 1, 4) AS year,
+        COUNT(CASE WHEN sales.price < 10000 THEN 1 END)
+            AS price_less_than_10k_count,
+        COUNT(CASE WHEN sales.price > 1000000 THEN 1 END)
+            AS price_greater_than_1m_count
+    FROM iasworld.sales
+    WHERE
+        sales.deactivat IS NULL
+        AND sales.cur = 'Y'
+        AND sales.price IS NOT NULL
+        AND sales.instruno IS NOT NULL
+        AND CAST(SUBSTR(sales.saledt, 1, 4) AS INTEGER) >= 2014
+        AND CAST(SUBSTR(sales.saledt, 1, 4) AS INTEGER) != 2500
```

Comment:
**[Question, non-blocking]** I'm guessing there's one weird sale with a mistyped date, is that right? If so, perhaps it would be better to leave in the erroneous sale, and attempt to fix the underlying data? I'm curious what @dfsnow thinks about this.

Reply:
Yup, there is an erroneous sale. The issue which removing it is trying to correct is that there are tests for decreases in X value, and if that remains, then it will always code 2023 - 2500 as having a sharp decrease.

Reply:
Woof, yeah I deactivated this sale in iasWorld since I can't find any trace of it in MyDec or elsewhere.